src/hooks/useIsHostIn/useIsHosetIn.test.tsx

```diff
@@ -0,0 +1,78 @@
+import { act, renderHook } from '@testing-library/react-hooks';
+import EventEmitter from 'events';
+import useDominantSpeaker from '../useDominantSpeaker/useDominantSpeaker';
+import useIsHosetIn from './useIsHostIn';
+import useVideoContext from '../useVideoContext/useVideoContext';
+import { NOTIFICATION_MESSAGE } from '../../utils/displayStrings';
+
+jest.mock('../useVideoContext/useVideoContext');
+jest.mock('../useDominantSpeaker/useDominantSpeaker');
+
+const onmsg = jest.fn(() => {});
+
+const mockedVideoContext = useVideoContext as jest.Mock<any>;
+
+describe('the useIsHosetIn hook', () => {
+  let mockRoom: any;
+
+  beforeEach(() => {
+    mockRoom = new EventEmitter();
+    mockedVideoContext.mockImplementation(() => ({
+      room: mockRoom,
+    }));
+  });
+
+  it('should return true by default', () => {
```

Comment:
seeing as this seems to test the case of when there are no participants yet can you change the description of the test